content/blog/2020/05/2020-05-18-read-only-jenkins-announcement.adoc

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Read-only Jenkins Configuration"
+tags:
+- system-read
+- extended-read
+- configuration-as-code
+- announcement
+author: timja
+---
+
+I'm excited to announce that the 'read-only' Jenkins feature is now available.
```

Comment:
```suggestion
I'm excited to announce that the 'read-only' Jenkins feature is now available for preview.
This feature allows restricting configuration UIs and APIs while providing access to essential Jenkins system configuration, diagnostics, and self-monitoring tools through Web UI.
Such mode is critical for instances managed as code, e.g. with plugin:configuration-as-code[Jenkins Configuration-as-Code] plugin.
It is delivered as a part of the jep:224[Readonly system configuration] effort.
```

Reply:
perfect thanks! I definitely meant to mention the jep!